champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/axe/Flash.java

```diff
@@ -54,12 +54,12 @@ public String getName() {
     @Override
     public String[] getDescription(int level) {
         return new String[] {
-                "Teleport a short distance horizontally",
-                "in the direction you are facing.",
+                "Right click with an Axe to activate",
                 "",
-                "Uses up to <val>" + getMaxCharges(level) + "</val> charges.",
+                "Teleport 5 Blocks in the direction you are facing",
+                "Stores up to <val>" + getMaxCharges(level) + "</val> charges",
                 "",
-                "Gain a charge every: <stat>" + getRechargeSeconds(level) + "</stat> seconds."
+                "Gain a charge every: <stat>" + (int)getRechargeSeconds(level) + "</stat> seconds"
```

Comment:
Casting to int only works if getRechargeSeconds is always guaranteed to be an int.  Which in config could be changed. _most_ times are represented as a double.

Reply:
I think it looks more consistent and better if all STATS are ints, if problems arise it can be fixed

Reply:
I mean, but they aren't. And you cannot guarantee it. One of the main reasons why I am putting as much into the config is so that it is much easier to change it in the future, and being able to see that update is important. Yes it looks messy, but unless we want to do logic to display a certain way (which is an option), it needs to be able to show the decimal because it _could_ be a decimal.

Reply:
Some statistic values need to be doubles, like cooldowns. In this specific scenario, the variable can be changed to be to a decimal number like @StudentAlleg said

Reply:
alright ill fix